app/learning/index.md

```diff
@@ -5,204 +5,98 @@ title: Getting started with Yeoman
 sidebar: sidebars/learning.md
 ---
 
-The Yeoman workflow is comprised of three core tools for improving your productivity and satisfaction when building a web app. These tools are:
+Yeoman is a generic scaffolding system allowing you to create any kind of app. It'll allow you to rapidly get started on new projects and streamline the maintenance of existing projects.
 
-* [yo](https://github.com/yeoman/yo) - the scaffolding tool from Yeoman
-* A package manager, such as [bower](http://bower.io/) or [npm](https://www.npmjs.com/).
-* A build tool, such as [Grunt](http://gruntjs.com/) or [Gulp](http://gulpjs.com/).
+Yeoman is language agnostic. It can generates projects in any languages (Web, Java, Python, C#, etc.)
```

Comment:
`Yeoman is language agnostic. It can generate projects in any language (Web, Java, Python, C#, etc.)`